applications/butler/values.yaml

```diff
@@ -64,13 +64,16 @@ global:
   vaultSecretsPath: ""
 
 config:
-  # -- URI to the DirectButler repository index file listing the configurations
-  # for each repository to be hosted by this server.
-  indexUri: ""
+  # -- Mapping from Butler repository label to Butler configuration URI for
+  # repositories which will be hosted by this server.
+  repositories: {}
 
-  # -- List of Butler repository labels which will be hosted by this server,
-  # matching those from the index file.
-  repositoryLabels: []
+  # -- Postgres username used to connect to the Butler DB
+  # @default -- Use values specified in per-repository Butler config files.
+  pguser: ""
+
+  # -- URL for the S3 service where files for datasets are stored by Butler.
+  s3_endpoint_url: ""
```

Comment:
The Helm convention is to use lowerCamelCase for all settings, so s3EndpointUrl. It produces a bit of unfortunate mismatch between Helm and Python in order to follow the conventions of both ecosystems in their relevant code.